esignet-default.properties

```diff
@@ -303,7 +303,7 @@ mosip.esignet.ui.config.key-values={'sbi.env': 'Developer', 'sbi.timeout.DISC':
   'sbi.capture.count.iris': 1, 'sbi.capture.score.face': 70, 'sbi.capture.score.finger':70, 'sbi.capture.score.iris':70, \
   'resend.otp.delay.secs': ${mosip.kernel.otp.expiry-time}, \
   'send.otp.channels' : '${mosip.esignet.authenticator.ida.otp-channels}', 'captcha.sitekey' : '${mosip.esignet.captcha-validator.site-key}', \
-  'captcha.enable' : 'otp', 'auth.txnid.length' : '${mosip.esignet.auth-txn-id-length}'}
+  'captcha.enable' : 'otp', 'auth.txnid.length' : '${mosip.esignet.auth-txn-id-length}','captcha.enable': ''}
```

Comment:
why captcha.enable is repeated ?

Reply:
I added it by mistake and corrected it now.